assets/dev/scss/editor/panel/panel.scss

```diff
@@ -127,5 +127,35 @@
 	}
 }
 
+// Notification Center (v1)
+@mixin e-notification-icon-bullet() {
+	position: relative;
+
+	&:after {
+		position: absolute;
+		content: '';
+		display: block;
+		background: var(--e-a-color-primary);
+		border-radius: 50%;
+		width: 8px;
+		height: 8px;
+		top: 5px;
+		right: 5px;
+	}
+}
+
+body.e-has-notification:not(.e-route-panel-menu) {
+	.elementor-panel-menu-item.elementor-panel-menu-item-notification-center {
```

Comment:
It would be more appropriate to use prefixes such as `e-`.

Reply:
@rodolphebertozzo It's a old classes.